docs/src/lib/sets/SparsePolynomialZonotope.md

```diff
@@ -0,0 +1,21 @@
+```@meta
+CurrentModule = LazySets
+```
+
+# [SparsePolynomialZonotope](@id def_SparsePolynomialZonotope)
+
+```@docs
+SparsePolynomialZonotope
+center(::SparsePolynomialZonotope)
+dependent_genmat(::SparsePolynomialZonotope)
+independent_genmat(::SparsePolynomialZonotope)
+expmat(::SparsePolynomialZonotope)
+indexvector(P::SparsePolynomialZonotope)
+dim(::SparsePolynomialZonotope)
+ndependentgens(::SparsePolynomialZonotope)
+nindependentgens(::SparsePolynomialZonotope)
+ngens(::SparsePolynomialZonotope)
```

Comment:
I wonder if we should have `genmat` as for `SimpleSparsePolynomialZonotope` and `AbstractZonotope`. Then if you `convert` such a set to an SPZ, the same function works.
As for `ngens` defined as the sum, this is indeed a generalization because there would be 0 independent generators, so the results coincide.

Reply:
I also wondered whether we should define `genmat` for `SparsePolynomialZonotope`, I thought a logical definition would be to have 

```julia
genmat(P::SparsePolynomialZonotope) = hcat(P.G, P.GI)
```

but I'm not sure whether this would be useful.

Since `SparsePolynomialZonotope` is not a subset of `AbstractZonotope`, I think the conversion to SPZ would still work?

Reply:
> Since `SparsePolynomialZonotope` is not a subset of `AbstractZonotope`, I think the conversion to SPZ would still work?

If anything, it would be the other way around.
This is not a problem, just slightly inconvenient that something does not work anymore that used to work before the conversion. I am also fine to not have `genmat` (or to add it later if we find it useful).

The concatenation would be consistent with `ngens`. I originally thought about just the matrix that is used when converting from a `Zonotope`, but I do not like the inconsistency with `ngens` then.

Reply:
added `genmat` to return the concatenation of the generator matrices

Reply:
I was thinking about it again and see only now that there is probably no application of the concatenation, sorry :sweat:
Let's see what @mforets says.